### PR TITLE
[BACKLOG-34864] Determine Maprfs storage scheme based on Shim Vendor

### PR DESCRIPTION
**Pull request by kodehash**
This change sets the storage scheme to maprfs in the NamedCluster. This change resonates across successful execution of Hadoop copy files directory structure and also with Hbase jobs.

### CODE REVIEW

File: kettle-plugins/hadoop-cluster/ui/src/main/java/org/pentaho/big/data/kettle/plugins/hadoopcluster/ui/endpoints/HadoopClusterManager.java [hunk 1]
```diff
@@ -212,6 +214,8 @@ public JSONObject importNamedCluster( ThinNameClusterModel model,
       nc.setName( model.getName() );
       nc.setHdfsUsername( model.getHdfsUsername() );
       nc.setHdfsPassword( nc.encodePassword( model.getHdfsPassword() ) );
+      if (MAPR_SHIM.equals(model.getShimVendor()))
```
Comment (lucboudreau) [inline kettle-plugins/hadoop-cluster/ui/src/main/java/org/pentaho/big/data/kettle/plugins/hadoopcluster/ui/endpoints/HadoopClusterManager.java:217]: Please use brackets even for single instructions in IF blocks.

File: kettle-plugins/hadoop-cluster/ui/src/main/java/org/pentaho/big/data/kettle/plugins/hadoopcluster/ui/endpoints/HadoopClusterManager.java [hunk 2]
```diff
@@ -260,6 +264,8 @@ private NamedCluster convertToNamedCluster( ThinNameClusterModel model ) {
     nc.setOozieUrl( model.getOozieUrl() );
     nc.setKafkaBootstrapServers( model.getKafkaBootstrapServers() );
     resolveShimIdentifier( nc, model.getShimVendor(), model.getShimVersion() );
+    if (MAPR_SHIM.equals(model.getShimVendor()))
```
Comment (lucboudreau) [inline kettle-plugins/hadoop-cluster/ui/src/main/java/org/pentaho/big/data/kettle/plugins/hadoopcluster/ui/endpoints/HadoopClusterManager.java:267]: Please use brackets even for single instructions in IF blocks.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
kettle-plugins/hadoop-cluster/ui
```
---
:ok_hand: All tests passed!
 
Tests run: **25**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2150/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by lucboudreau — CHANGES_REQUESTED** [review]

**Comment by kodehash** [comment]
Closing this PR as a new one is created from a forked Repo.

**Pull request closed without merging**